.circleci/scripts/binary_upload.sh

```diff
@@ -37,13 +37,22 @@ do_backup() {
 conda_upload() {
   (
     set -x
-    ${ANACONDA} \
+    #Loop until counter is 3
+    counter=1
+    while [[ $counter -le 3 ]]
+    do
+      ${ANACONDA} \
       upload  \
       ${PKG_DIR}/*.tar.bz2 \
       -u "pytorch-${UPLOAD_CHANNEL}" \
       --label main \
       --no-progress \
       --force
+
+      rc=$?
+      # In case of failure, do a retry upload
+      [[ $rc -ne 0 ]] && ((counter++)) || break
```

Comment:
Could you please explain/clarify what condition would trigger break?  :)

Reply:
sure break will execute on rc==0 which is success